2.0/SciFiHelmet/README.md

```diff
@@ -0,0 +1,11 @@
+# Sci Fi Helmet
+## Screenshot
+
+![screenshot](screenshot/screenshot.jpg)
+
+
+## License Information
+
+["Sci Fi Helmet" by Michael Pavlovic](http://quixel.se/usermanual/quixelsuite/doku.php?id=ddo_samples)
```

Comment:
At quick glance, I'm not sure what the license is.  Do you know or can you confirm with the artist that we can use it here?